build.js

```diff
@@ -37,6 +37,8 @@ const anchorMarkdownHeadings = require('./scripts/plugins/anchor-markdown-headin
 const loadVersions = require('./scripts/load-versions')
 const latestVersion = require('./scripts/helpers/latestversion')
 
+const banner = require('./locale/en/site.json').banner
```

Comment:
There is the `i18nJSON` function that returns the value or the default

Reply:
oof, maybe i should read first lol